File: course_101/exercises/easy_2/when_will_i_retire.rb
puts "What is your age?"
age = gets.to_i

puts "When would you like to retire?"
retirement_age = gets.to_i

current_year = Time.now.year
years_till_retirement = retirement_age - age
year_of_retirement = current_year + years_till_retirement


puts "It's #{current_year}. You will retire in #{year_of_retirement}"
puts "You only have #{years_till_retirement} years of work to go!"

File: course_js210_computational_thinking_and_javascript_programming/exercises/js210_small_problems/easy_3/when_will_i_retire.js
// Build a program that logs when the user will retire and how many more years the user has to work until retirement.

let readlinesSync = require('readline-sync').question;

let age = Number(readlinesSync('What is your age? '));
let retirementAge = Number(readlinesSync('At what age would you like to retire? '));

let currentYear = new Date().getFullYear();

console.log(`It's ${currentYear}. You will retire in ${retirementAge - age + currentYear}`);
console.log(`You only have ${retirementAge - age} of work to go!`);